docs/standard/basetypes/customnumeric.md

```diff
@@ -0,0 +1,645 @@
+---
```

Comment:
Look at the SEO material I've sent from the writer's forum last week or something. file names are important too and words should be separated by a dash. I'd review all file names in this PR to make sure they help with SEO.